2996_reflection/reflection.md

```diff
@@ -271,4 +397,172 @@ std::string name1 = name_of(^S);             // Also okay.
 
 ## Metafunctions
 
+We propose a number of metafunctions declared in namespace `std::meta` to operator on reflection values.
+Adding metafunctions to an implementation is expected to be relatively "easy" compared to implementing the core language features described previously.
+However, despite offering a normal consteval C++ function interface, each on of these relies on "compiler magic" to a significant extent.
+
+### `invalid_reflection`, `is_invalid`, `diagnose_error`
+
+:::bq
+```c++
+consteval auto invalid_reflection(
+                  std::string_view message,
+                  std::source_location src_loc = std::source_location::current())->info;
+consteval auto is_invalid(info)->bool;
+consteval auto is_invalid(std::span<info>)->bool;
+consteval void diagnose_error(info);
```

Comment:
I think for these for consistency/clarity present these with the namespace. That way we don't need all the extra `std::`s and since the namespace is _right there_ anyway, should be easy enough to understand (ditto throughout):
```suggestion
namespace std::meta {
  consteval auto invalid_reflection(
                  string_view message,
                  source_location src_loc = source_location::current()) -> info;
  consteval auto is_invalid(info) -> bool;
  consteval auto is_invalid(span<info>) -> bool;
  consteval auto diagnose_error(info) -> void;
}
```
Also, separately - so we have `is_invalid(info)` and... `is_invalid(span<info>)`? I don't think we need the other one. I can already write `std::ranges::any_of(stuff, std::meta::is_invalid)`.